operator/Makefile

```diff
@@ -410,7 +410,7 @@ upgrade-dirty-tag-via-olm: kuttl
 # Commands to enter local Python virtual environment and get needed dependencies there.
 ACTIVATE_PYTHON = python3 -m venv bundle_helpers/.venv ;\
 	. bundle_helpers/.venv/bin/activate ;\
-	pip3 install --upgrade pip==21.3.1 setuptools==59.6.0 ;\
```

Comment:
Omitting versions here may lead to problems when maintainers release broken versions. Omitting versions also leads to non-reproducible builds.

I think, we ran into something like this which forced us put versions. If the current version numbers are too old, you can try put some newer ones from https://pypi.org/

Reply:
I've bumped the version